shell/build-jsonnet.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
How about instead of making this it's own file we make this `render_jsonnet` function so we can use it in multiple places?

Reply:
I need to call it as a script

Reply:
it also doesn't help anything much

Reply:
Alright, though if we have another script that essentially calls this like a fn imma blame you :P